cross/tcl/Makefile

```diff
@@ -2,33 +2,74 @@ PKG_NAME = tcl
 PKG_VERS = 8.6.6
 PKG_EXT = tar.gz
 PKG_DIST_NAME = $(PKG_NAME)$(PKG_VERS)-src.$(PKG_EXT)
-PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl/
+PKG_DIST_SITE = http://prdownloads.sourceforge.net/tcl
 PKG_DIR = $(PKG_NAME)$(PKG_VERS)/unix
 
 DEPENDS =
 BUILD_DEPENDS = cross/zlib
```

Comment:
Move to DEPENDS, unless you're 100% certain that zlib is only a compile-time dependency (docs suggesting that would be helpful here)